source/includes/minimum-destination-privileges.rst

```diff
@@ -0,0 +1,112 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 10 20
+
+   * - Sync Type
+     - Minimum Destination Privileges
+
+   * - Default
+     - .. code-block:: javascript
+       
+          [
```

Comment:
Let's reduce the indentation to three spaces at every level. The note on the next section links out to a gist that shows what I mean about the spacing.